pkgs/top-level/all-packages.nix

```diff
@@ -26839,4 +26839,6 @@ in
   vpsfree-client = callPackage ../tools/virtualization/vpsfree-client {};
 
   gpio-utils = callPackage ../os-specific/linux/kernel/gpio-utils.nix { };
+
+  utahfs = callPackage ../applications/networking/utahfs { };
```

Comment:
Please, don't add to the end of the file. Instead, I suggest you locate an appropriate section (e.g. using `grep '###' pkgs/top-level/all-packages.nix`, probably `### TOOLS` or `### APPLICATIONS`) and insert your attribute according to the alphabetic order. This should also make merge conflicts unlikely.

Reply:
Done. Thak you for the review!